mono/mini/interp/interp.c

```diff
@@ -5487,6 +5589,9 @@ mono_ee_interp_init (const char *opts)
 	mono_interp_transform_init ();
 
 	MonoEECallbacks c;
+#ifdef MONO_ARCH_HAVE_INTERP_ENTRY_TRAMPOLINE
+	c.entry_from_trampoline = (void (*) (gpointer, gpointer)) interp_entry_from_trampoline;
```

Comment:
nit: that's a bit ugly, I think `interp_entry_from_trampoline` should have the same signature as `c.entry_from_trampoline`, i.e. move the casts inside `interp_entry_from_trampoline`

Reply:
@lewurm I'll fix it in the next PR so I don't restart CI for the 10th time